numpy/linalg/linalg.py

```diff
@@ -1382,7 +1382,7 @@ def cond(x, p=None):
 
     Parameters
     ----------
-    x : (M, N) array_like
+    x : (..., M, N) array_like
```

Comment:
Sorry. But things like: http://icl.cs.utk.edu/lapack-forum/archives/lapack/msg00311.html

It seems to me that the definition with the pseudo inverse is definitely useful makes sense. So maybe we should add another sentence or two to the Notes to say that. Frankly, I am not quite sure what to write, but I can think about it later.